diff --git a/node_modules/react-native-scrollable-tab-view/Button.web.js b/node_modules/react-native-scrollable-tab-view/Button.web.js
new file mode 100644
index 0000000..c2cc1a4
--- /dev/null
+++ b/node_modules/react-native-scrollable-tab-view/Button.web.js
@@ -0,0 +1,14 @@
+const React = require('react');
+const ReactNative = require('react-native');
+const {
+ TouchableOpacity,
+  View,
+} = ReactNative;
+
+const Button = (props) => {
+  return <TouchableOpacity {...props}>
+    {props.children}
+  </TouchableOpacity>;
+};
+
+module.exports = Button;
diff --git a/node_modules/react-native-scrollable-tab-view/index.js b/node_modules/react-native-scrollable-tab-view/index.js
index af94d86..d6b45b5 100644
--- a/node_modules/react-native-scrollable-tab-view/index.js
+++ b/node_modules/react-native-scrollable-tab-view/index.js
@@ -127,7 +127,7 @@ const ScrollableTabView = createReactClass({
   },
 
   goToPage(pageNumber) {
-    if (Platform.OS === 'ios') {
+    if (Platform.OS === 'ios' || Platform.OS === "web") {
       const offset = pageNumber * this.state.containerWidth;
       if (this.scrollView) {
         this.scrollView.getNode().scrollTo({x: offset, y: 0, animated: !this.props.scrollWithoutAnimation, });
@@ -216,7 +216,7 @@ const ScrollableTabView = createReactClass({
   },
 
   renderScrollableContent() {
-    if (Platform.OS === 'ios') {
+    if (Platform.OS === 'ios' || Platform.OS === 'web') {
       const scenes = this._composeScenes();
       return <Animated.ScrollView
         horizontal
